a3p-integration/proposals/n:upgrade-next/test-lib/vats.js

```diff
@@ -0,0 +1,18 @@
+import { getDetailsMatchingVats } from '@agoric/synthetic-chain';
+import { naturalCompare } from '@agoric/internal/src/natural-sort.js';
+
+export const getGovernedVatDetails = async (nameCommon, includeTerminated) => {
+  const detailsArray = await getDetailsMatchingVats(nameCommon);
```

Comment:
exporting `getGovernedVatDetails` adds another case of exported a function that closes over ambient authority (`better-sqlite3`).

Didn't I refactor [vat-status.js](https://github.com/Agoric/agoric-3-proposals/blob/main/packages/synthetic-chain/src/lib/vat-status.js) to accept explicitly injecting the relevant authority? Darn it; I guess I didn't land that.

OK, I'll just add this to the back-links for...

 - #2160